src/components/ui/Logo/Logo.tsx

```diff
@@ -1,145 +1,6 @@
-        fill="#1A3470"
-      />
-      <defs>
-        <linearGradient
-          id="paint0_linear_2299_155683"
-          x1="12"
-          y1="20.5"
-          x2="40.5"
-          y2="4"
-          gradientUnits="userSpaceOnUse"
-        >
-          <stop offset="0.0677083" stopColor="#1C3670" />
-          <stop offset="1" stopColor="#B6C6ED" />
-        </linearGradient>
-        <linearGradient
-          id="paint1_linear_2299_155683"
-          x1="19"
-          y1="-4.02331e-07"
-          x2="32.5"
-          y2="-9.5"
-          gradientUnits="userSpaceOnUse"
-        >
-          <stop stopColor="#001947" />
-          <stop offset="1" stopColor="#001947" stopOpacity="0" />
-        </linearGradient>
-        <linearGradient
-          id="paint2_linear_2299_155683"
-          x1="15.569"
-          y1="13.9135"
-          x2="20.361"
-          y2="19.778"
-          gradientUnits="userSpaceOnUse"
-        >
-          <stop stopColor="#9FB8EF" />
-          <stop offset="0.934068" stopColor="#0A1C46" />
-        </linearGradient>
-        <linearGradient
-          id="paint3_linear_2299_155683"
-          x1="16"
-          y1="30.5"
-          x2="22.3667"
-          y2="38.2918"
-          gradientUnits="userSpaceOnUse"
-        >
-          <stop stopColor="#9FB8EF" />
-          <stop offset="0.934068" stopColor="#0A1C46" />
-        </linearGradient>
-      </defs>
-    </svg>
-  )
+  // eslint-disable-next-line @next/next/no-img-element
+  return <img src="/logo.svg" alt="logo" width="176px" height="30px" />
```

Comment:
Found this better alt text inside the previous SVG:
```suggestion
  return <img src="/logo.svg" alt="The store logo with with a stylized letter B in a dark blue gradient
        background" width="176px" height="30px" />
```